mpc/mpc-aio/src/protocol/ot/kos/receiver.rs

```diff
@@ -110,6 +115,50 @@ impl ObliviousReceive<bool, Block> for Kos15IOReceiver<r_state::RandSetup> {
     }
 }
 
+// The idea is to send AES encryption keys in the OT, which can then later be used by the receiver
```

Comment:
// The idea for obliviously receiving a vec of Blocks is for the sender to send an AES encryption key
// using OT, which can then later be used by the receiver to decrypt an arbitrary long message, which
// is sent shortly after the OT. This way we extend our OT from 128-bit maximum message length to an
// unlimited message length.